src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int RANDOM_RA = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureIndex = random.nextInt(4);
+        switch (figureIndex) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            default:
+                return getRandomSquare();
+        }
+    }
+
+    private Figure getRandomSquare() {
+        return new Square(colorSupplier.getRandomColor(), getRandomInt());
+    }
+
+    private Figure getRandomRightTriangle() {
+        return new RightTriangle(colorSupplier.getRandomColor(),
+                        getRandomInt(),
+                        getRandomInt());
+    }
+
+    private Figure getRandomRectangle() {
+        return new Rectangle(colorSupplier.getRandomColor(),
+                        getRandomInt(),
+                        getRandomInt());
+    }
+
+    private Figure getRandomIsoscelesTrapezoid() {
+        return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        getRandomInt(),
+                        getRandomInt(),
+                        getRandomInt());
+    }
+
+    private Figure getRandomCircle() {
+        return new Circle(colorSupplier.getRandomColor(), getRandomInt());
+    }
+
+    private int getRandomInt() {
+        return random.nextInt(RANDOM_RA);
+    }
+
+    public Figure getDefaultFigure() {
+        Circle whiteCircle = new Circle("White", 10);
+        return whiteCircle;
```

Comment:
```suggestion
        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
```
lets create constants for "white" annd 10 since those are default values

Reply:
you have not fixed this